src/service/impl/journey.ts

```diff
@@ -3,27 +3,17 @@ import { IJourneyService } from '../interface';
 import { Result } from '@badrap/result';
 import { APIError, TripPatternsQuery } from '../types';
 
-import { PubSub } from '@google-cloud/pubsub';
-import { getEnv } from '../../utils/getenv';
 import { generateId, getServiceIds } from '../../utils/journey-utils';
 import { EnturServiceAPI } from './entur';
 
-const ENV = getEnv();
-const topicName = `analytics_trip_search`;
+interface journeyAnalyticsPublisher {
```

Comment:
Burde ikke dette være typer eksportert fra Analytics for å enklere gjøre refaktorisering og reflektere sannheten, eller er tanken eksplisitt API fra konsument-perspektivet som skal brekke ved endringer med å benytte seg av strukturelle typer?

Reply:
Dette var ment som dependency inversion av infrastruktur. Jeg tror ikke noe på "utsiden" vil kreve en eksportert type.